poprox-db/migrations/versions/2024_11_04_1040-ac1a317899d9_add_survey_code_column_to_qualtrics_.py

```diff
@@ -0,0 +1,26 @@
+"""add survey_code column to qualtrics_surveys table
+
+Revision ID: ac1a317899d9
+Revises: 8f10a2101cb8
+Create Date: 2024-11-04 10:40:23.404585
+
+"""
+
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision: str = "ac1a317899d9"
+down_revision: Union[str, None] = "8f10a2101cb8"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    op.add_column("qualtrics_surveys", sa.Column("survey_code", sa.String(length=3), nullable=True))
```

Comment:
I think we may as well use the existing survey code field from the spreadsheet, so I added it here and we can manually insert the codes in the existing survey rows

Reply:
Should we drop the length=3 limit? I don't know if that does us much good in practice since I think Shahan mentioned already running short on 3-letter-codes that are useful.